eng/Versions.props

```diff
@@ -27,6 +27,6 @@
     <SystemDataSqlClientVersion>4.8.0</SystemDataSqlClientVersion>
     <MicrosoftNETCoreAppInternalPackageVersion>3.1.0-rtm.19565.2</MicrosoftNETCoreAppInternalPackageVersion>
     <MicrosoftNETCoreAppRefPackageVersion>3.1.0</MicrosoftNETCoreAppRefPackageVersion>
-    <MicrosoftNETCoreAppRuntimewinx64PackageVersion>3.1.0</MicrosoftNETCoreAppRuntimewinx64PackageVersion>
+    <MicrosoftNETCoreAppRuntimewinx64PackageVersion>3.1.1</MicrosoftNETCoreAppRuntimewinx64PackageVersion>
```

Comment:
@JunTaoLuo should this be updated as well?

Reply:
I think so cc @ajcvickers @bricelam

Reply:
Seems fine.